src/index.html

```diff
@@ -5,8 +5,395 @@
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#"
+        class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="MOYO logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple"
+              class="nav__link is-active"
+            >
+            Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung"
+              class="nav__link"
+            >
+            Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones"
+              class="nav__link"
+            >
+            Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops&computers"
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets"
+              class="nav__link"
+            >
+            Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets"
+              class="nav__link"
+            >
+            Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo"
+              class="nav__link"
+            >
+            Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#video"
+              class="nav__link"
+            >
+            Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="Apple"
+          class="card__image"
+        >
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__product-code">
+          Product code: 195434
+        </p>
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review-value">
+            Reviews: 5
+          </p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-name">
+            Price:
+          </p>
+          <p class="card__price-value">
+            $2,199
+          </p>
+        </div>
+        <a href="#"
+          class="card__buttom"
+          data-qa="card__hover"
+       >
+        Buy
+        </a>
```

Comment:
Need to check and fix it everywhere
```suggestion
        <a
          href="#"
          class="card__buttom"
          data-qa="card__hover"
       >
         Buy
        </a>
```